src/libraries/tests.proj

```diff
@@ -99,6 +99,9 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json\tests\System.Text.Json.SourceGeneration.Unit.Tests\System.Text.Json.SourceGeneration.Roslyn4.0.Unit.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json\tests\System.Text.Json.SourceGeneration.Tests\System.Text.Json.SourceGeneration.Roslyn3.11.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json\tests\System.Text.Json.SourceGeneration.Tests\System.Text.Json.SourceGeneration.Roslyn4.0.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json\tests\System.Text.Json.SourceGeneration.Unit.Tests\System.Text.Json.SourceGeneration.Roslyn4.4.Unit.Tests.csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json\tests\System.Text.Json.SourceGeneration.Tests\System.Text.Json.SourceGeneration.Roslyn4.4.Tests.csproj" />
```

Comment:
These are the same projects as the `4.0.Tests`, they were just renamed in https://github.com/dotnet/runtime/pull/79828. We should either use regex to match the projects via something like `System.Text.Json\tests\System.Text.Json.SourceGeneration.*\**\*.csproj` or disable the tests in the projects themselves as suggested by Alex, similar to https://github.com/dotnet/runtime/blob/6251d496e307ed39f1e600cd036106cc1bd7e340/src/libraries/System.Drawing.Common/tests/System.Drawing.Common.Tests.csproj#L6